infrastructure/user-list-search/sqs_sns_topic_subscription.tf

```diff
@@ -19,6 +28,30 @@ resource "aws_sns_topic_subscription" "user_events_sns_topic_subscription" {
   depends_on = [aws_sqs_queue.user_events_sns_topic_dlq, aws_sqs_queue.user_list_events]
 }
 
+resource "aws_sns_topic_subscription" "corpus_events_sns_topic_subscription" {
+  topic_arn = local.corpusEventsSnsTopicArn
+  protocol  = "sqs"
+  endpoint  = aws_sqs_queue.corpus_events.arn
+  # The version of terraform used in this service does not support redrive_policy
+  # an update is required. It is not blocking but should be prioritized soon
+  #  redrive_policy = jsonencode({
+  #    deadLetterTargetArn: aws_sqs_queue.corpus_events_sns_topic_dlq.arn
+  #  })
+  depends_on = [aws_sqs_queue.corpus_events_sns_topic_dlq, aws_sqs_queue.corpus_events]
+}
+
+resource "aws_sns_topic_subscription" "collection_events_sns_topic_subscription" {
+  topic_arn = local.collectionEventsSnsTopicArn
+  protocol  = "sqs"
+  endpoint  = aws_sqs_queue.corpus_events.arn
+  # The version of terraform used in this service does not support redrive_policy
```

Comment:
Hmm.. this should be on 1.8..

Reply:
I copy-pasted this, maybe can be removed?